### DISCUSSION TITLE
Upgraded to 2.3.40 - Strelka Containers in ERROR

### DISCUSSION THREAD

**Discussion by apmitche199** [Security-Onion-Solutions/securityonion#3673] [Unsupported Versions]
Recently Upgraded to 2.3.40 from 2.3.30 

All other containers start with no issue

However have
    so-strelka-backend ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ ERROR ]    
    so-strelka-coordinator --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ ERROR ]    
    so-strelka-filestream ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ ERROR ]    
    so-strelka-frontend ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ [ ERROR ]    
    so-strelka-gatekeeper ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ ERROR ]    
    so-strelka-manager ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ ERROR ] 

When running

sudo so-strelka-start;

[INFO    ] Running state [netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0] at time 10:00:13.614988
[INFO    ] Executing state cmd.run for [netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0]
[INFO    ] Executing command 'netstat' in directory '/home/apmitche'
[INFO    ] {'pid': 20908, 'retcode': 0, 'stdout': '', 'stderr': ''}
[INFO    ] Completed state [netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0] at time 10:00:13.800355 (duration_in_ms=185.368)

If I run 

sudo netstat -utanp | grep ":57314"

I do not see any output 

Appreciate any guidance - thanks

**Comment by weslambert** [comment]
I don't see an error here.  I believe the text you are seeing is just the command that is run to check if another process is already, not the actual output of the command being run.  Have you tried rebooting, or running `so-strelka-restart` to remove the older containers?

**Reply by apmitche199** [reply]
Hi Wes

Here is the output of so-strelka-restart and so-status

$ sudo so-strelka-restart
=========================================================================
Stopping strelka-filestream...
=========================================================================
so-strelka-filestream
so-strelka-filestream
=========================================================================
Stopping strelka-manager...
=========================================================================
so-strelka-manager
so-strelka-manager
=========================================================================
Stopping strelka-frontend...
=========================================================================
so-strelka-frontend
so-strelka-frontend
=========================================================================
Stopping strelka-backend...
=========================================================================
so-strelka-backend
so-strelka-backend
=========================================================================
Stopping strelka-gatekeeper...
=========================================================================
so-strelka-gatekeeper
so-strelka-gatekeeper
=========================================================================
Stopping strelka-coordinator...
=========================================================================
so-strelka-coordinator
so-strelka-coordinator
=========================================================================
Starting strelka...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
[INFO    ] Loading fresh modules for state activity
[INFO    ] Running state [/opt/so/conf/strelka] at time 06:36:01.023722
[INFO    ] Executing state file.directory for [/opt/so/conf/strelka]
[INFO    ] The directory /opt/so/conf/strelka is in the correct state
[INFO    ] Completed state [/opt/so/conf/strelka] at time 06:36:01.938622 (duration_in_ms=914.901)
[INFO    ] Running state [/opt/so/conf/strelka/rules] at time 06:36:01.938982
[INFO    ] Executing state file.directory for [/opt/so/conf/strelka/rules]
[INFO    ] The directory /opt/so/conf/strelka/rules is in the correct state
[INFO    ] Completed state [/opt/so/conf/strelka/rules] at time 06:36:01.942976 (duration_in_ms=3.994)
[INFO    ] Running state [/opt/so/conf/strelka/] at time 06:36:01.943324
[INFO    ] Executing state file.recurse for [/opt/so/conf/strelka/]
[INFO    ] The directory /opt/so/conf/strelka/ is in the correct state
[INFO    ] Completed state [/opt/so/conf/strelka/] at time 06:36:02.209948 (duration_in_ms=266.624)
[INFO    ] Running state [/opt/so/conf/strelka/rules] at time 06:36:02.210311
[INFO    ] Executing state file.recurse for [/opt/so/conf/strelka/rules]
[INFO    ] The directory /opt/so/conf/strelka/rules is in the correct state
[INFO    ] Completed state [/opt/so/conf/strelka/rules] at time 06:36:07.107580 (duration_in_ms=4897.269)
[INFO    ] Running state [/opt/so/saltstack/default/salt/strelka/rules/repos.txt] at time 06:36:07.107956
[INFO    ] Executing state file.managed for [/opt/so/saltstack/default/salt/strelka/rules/repos.txt]
[INFO    ] File /opt/so/saltstack/default/salt/strelka/rules/repos.txt is in the correct state
[INFO    ] Completed state [/opt/so/saltstack/default/salt/strelka/rules/repos.txt] at time 06:36:07.121167 (duration_in_ms=13.211)
[INFO    ] Running state [/nsm/strelka] at time 06:36:07.121429
[INFO    ] Executing state file.directory for [/nsm/strelka]
[INFO    ] The directory /nsm/strelka is in the correct state
[INFO    ] Completed state [/nsm/strelka] at time 06:36:07.123292 (duration_in_ms=1.863)
[INFO    ] Running state [/nsm/strelka/log] at time 06:36:07.123496
[INFO    ] Executing state file.directory for [/nsm/strelka/log]
[INFO    ] The directory /nsm/strelka/log is in the correct state
[INFO    ] Completed state [/nsm/strelka/log] at time 06:36:07.125293 (duration_in_ms=1.796)
[INFO    ] Running state [/nsm/strelka/processed] at time 06:36:07.125490
[INFO    ] Executing state file.directory for [/nsm/strelka/processed]
[INFO    ] The directory /nsm/strelka/processed is in the correct state
[INFO    ] Completed state [/nsm/strelka/processed] at time 06:36:07.127236 (duration_in_ms=1.745)
[INFO    ] Running state [/nsm/strelka/staging] at time 06:36:07.127431
[INFO    ] Executing state file.directory for [/nsm/strelka/staging]
[INFO    ] The directory /nsm/strelka/staging is in the correct state
[INFO    ] Completed state [/nsm/strelka/staging] at time 06:36:07.129461 (duration_in_ms=2.031)
[INFO    ] Running state [/nsm/strelka/unprocessed] at time 06:36:07.129656
[INFO    ] Executing state file.directory for [/nsm/strelka/unprocessed]
[INFO    ] The directory /nsm/strelka/unprocessed is in the correct state
[INFO    ] Completed state [/nsm/strelka/unprocessed] at time 06:36:07.131373 (duration_in_ms=1.717)
[INFO    ] Running state [netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0] at time 06:36:07.132154
[INFO    ] Executing state cmd.run for [netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0]
[INFO    ] Executing command 'netstat' in directory '/home/apmitche'
[INFO    ] {'pid': 7098, 'retcode': 0, 'stdout': '', 'stderr': ''}
[INFO    ] Completed state [netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0] at time 06:36:07.328590 (duration_in_ms=196.436)
[INFO    ] Running state [so-strelka-coordinator] at time 06:36:07.390792
[INFO    ] Executing state docker_container.running for [so-strelka-coordinator]
[INFO    ] {'container_id': {'added': 'b1aed5fc5e1703cce6f93ec1356c9803465fdb2aa9d6ecb3bb81a1ca01261785'}, 'state': {'old': None, 'new': 'running'}}
[INFO    ] Completed state [so-strelka-coordinator] at time 06:36:09.868997 (duration_in_ms=2478.206)
[INFO    ] Running state [/opt/so/conf/so-status/so-status.conf] at time 06:36:09.869471
[INFO    ] Executing state file.append for [/opt/so/conf/so-status/so-status.conf]
[INFO    ] File /opt/so/conf/so-status/so-status.conf is in correct state
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 06:36:09.884664 (duration_in_ms=15.193)
[INFO    ] Running state [so-strelka-gatekeeper] at time 06:36:09.884925
[INFO    ] Executing state docker_container.running for [so-strelka-gatekeeper]
[INFO    ] {'container_id': {'added': '0ddf29755aa5be6b76335e00ae5cba0312667289b37331b3108b3ff0624a96d0'}, 'state': {'old': None, 'new': 'running'}}
[INFO    ] Completed state [so-strelka-gatekeeper] at time 06:36:11.999921 (duration_in_ms=2114.996)
[INFO    ] Running state [/opt/so/conf/so-status/so-status.conf] at time 06:36:12.000270
[INFO    ] Executing state file.append for [/opt/so/conf/so-status/so-status.conf]
[INFO    ] File /opt/so/conf/so-status/so-status.conf is in correct state
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 06:36:12.003822 (duration_in_ms=3.552)
[INFO    ] Running state [so-strelka-frontend] at time 06:36:12.004061
[INFO    ] Executing state docker_container.running for [so-strelka-frontend]
[INFO    ] {'container_id': {'added': '4d7e054adef9a9e459f292a6e57c39d32b71a27da631df5b507cc7696bb97e08'}, 'state': {'old': None, 'new': 'running'}}
[INFO    ] Completed state [so-strelka-frontend] at time 06:36:13.800458 (duration_in_ms=1796.396)
[INFO    ] Running state [/opt/so/conf/so-status/so-status.conf] at time 06:36:13.800801
[INFO    ] Executing state file.append for [/opt/so/conf/so-status/so-status.conf]
[INFO    ] File /opt/so/conf/so-status/so-status.conf is in correct state
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 06:36:13.804156 (duration_in_ms=3.356)
[INFO    ] Running state [so-strelka-backend] at time 06:36:13.804355
[INFO    ] Executing state docker_container.running for [so-strelka-backend]
[INFO    ] {'container_id': {'added': '2e22e9eddb1fb6c7aebf043e68ed643847590ccd0be58da3128266af56b1fb58'}, 'state': {'old': None, 'new': 'running'}}
[INFO    ] Completed state [so-strelka-backend] at time 06:36:15.541115 (duration_in_ms=1736.759)
[INFO    ] Running state [/opt/so/conf/so-status/so-status.conf] at time 06:36:15.541501
[INFO    ] Executing state file.append for [/opt/so/conf/so-status/so-status.conf]
[INFO    ] File /opt/so/conf/so-status/so-status.conf is in correct state
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 06:36:15.544631 (duration_in_ms=3.13)
[INFO    ] Running state [so-strelka-manager] at time 06:36:15.544847
[INFO    ] Executing state docker_container.running for [so-strelka-manager]
[INFO    ] {'container_id': {'added': 'b6132364d01b196ee992e7d7b5f1785b387eca1e51b643417929ac0ddd723223'}, 'state': {'old': None, 'new': 'running'}}
[INFO    ] Completed state [so-strelka-manager] at time 06:36:17.800388 (duration_in_ms=2255.541)
[INFO    ] Running state [/opt/so/conf/so-status/so-status.conf] at time 06:36:17.802652
[INFO    ] Executing state file.append for [/opt/so/conf/so-status/so-status.conf]
[INFO    ] File /opt/so/conf/so-status/so-status.conf is in correct state
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 06:36:17.816466 (duration_in_ms=13.814)
[INFO    ] Running state [so-strelka-filestream] at time 06:36:17.816684
[INFO    ] Executing state docker_container.running for [so-strelka-filestream]
[INFO    ] {'container_id': {'added': '3170fe5bbb909e3256880709149bb87fcda76bb581a081ae745eaee865e51890'}, 'state': {'old': None, 'new': 'running'}}
[INFO    ] Completed state [so-strelka-filestream] at time 06:36:19.484785 (duration_in_ms=1668.1)
[INFO    ] Running state [/opt/so/conf/so-status/so-status.conf] at time 06:36:19.485123
[INFO    ] Executing state file.append for [/opt/so/conf/so-status/so-status.conf]
[INFO    ] File /opt/so/conf/so-status/so-status.conf is in correct state
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 06:36:19.488438 (duration_in_ms=3.314)
[INFO    ] Running state [[ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/ > /dev/null 2>&1] at time 06:36:19.491497
[INFO    ] Executing state cron.absent for [[ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/ > /dev/null 2>&1]
[INFO    ] Executing command 'crontab' in directory '/home/apmitche'
[INFO    ] Cron [ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/ > /dev/null 2>&1 already absent
[INFO    ] Completed state [[ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/ > /dev/null 2>&1] at time 06:36:19.498900 (duration_in_ms=7.402)[INFO    ] Running state [[ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/unprocessed/ > /dev/null 2>&1] at time 06:36:19.499285
[INFO    ] Executing state cron.present for [[ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/unprocessed/ > /dev/null 2>&1]
[INFO    ] Executing command 'crontab' in directory '/home/apmitche'
[INFO    ] Cron [ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/unprocessed/ > /dev/null 2>&1 already present
[INFO    ] Completed state [[ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/unprocessed/ > /dev/null 2>&1] at time 06:36:19.508261 (duration_in_ms=8.976)
local:
----------
          ID: strelkaconfdir
    Function: file.directory
        Name: /opt/so/conf/strelka
      Result: True
     Comment: The directory /opt/so/conf/strelka is in the correct state
     Started: 06:36:01.023721
    Duration: 914.901 ms
     Changes:
----------
          ID: strelkarulesdir
    Function: file.directory
        Name: /opt/so/conf/strelka/rules
      Result: True
     Comment: The directory /opt/so/conf/strelka/rules is in the correct state
     Started: 06:36:01.938982
    Duration: 3.994 ms
     Changes:
----------
          ID: strelkasync
    Function: file.recurse
        Name: /opt/so/conf/strelka/
      Result: True
     Comment: The directory /opt/so/conf/strelka/ is in the correct state
     Started: 06:36:01.943324
    Duration: 266.624 ms
     Changes:
----------
          ID: strelkarules
    Function: file.recurse
        Name: /opt/so/conf/strelka/rules
      Result: True
     Comment: The directory /opt/so/conf/strelka/rules is in the correct state
     Started: 06:36:02.210311
    Duration: 4897.269 ms
     Changes:
----------
          ID: strelkarepos
    Function: file.managed
        Name: /opt/so/saltstack/default/salt/strelka/rules/repos.txt
      Result: True
     Comment: File /opt/so/saltstack/default/salt/strelka/rules/repos.txt is in the correct state
     Started: 06:36:07.107956
    Duration: 13.211 ms
     Changes:
----------
          ID: strelkadatadir
    Function: file.directory
        Name: /nsm/strelka
      Result: True
     Comment: The directory /nsm/strelka is in the correct state
     Started: 06:36:07.121429
    Duration: 1.863 ms
     Changes:
----------
          ID: strelkalogdir
    Function: file.directory
        Name: /nsm/strelka/log
      Result: True
     Comment: The directory /nsm/strelka/log is in the correct state
     Started: 06:36:07.123497
    Duration: 1.796 ms
     Changes:
----------
          ID: strelkaprocessed
    Function: file.directory
        Name: /nsm/strelka/processed
      Result: True
     Comment: The directory /nsm/strelka/processed is in the correct state
     Started: 06:36:07.125491
    Duration: 1.745 ms
     Changes:
----------
          ID: strelkastaging
    Function: file.directory
        Name: /nsm/strelka/staging
      Result: True
     Comment: The directory /nsm/strelka/staging is in the correct state
     Started: 06:36:07.127430
    Duration: 2.031 ms
     Changes:
----------
          ID: strelkaunprocessed
    Function: file.directory
        Name: /nsm/strelka/unprocessed
      Result: True
     Comment: The directory /nsm/strelka/unprocessed is in the correct state
     Started: 06:36:07.129656
    Duration: 1.717 ms
     Changes:
----------
          ID: strelkaportavailable
    Function: cmd.run
        Name: netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0
      Result: True
     Comment: Command "netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0" run
     Started: 06:36:07.132154
    Duration: 196.436 ms
     Changes:
              ----------
              pid:
                  7098
              retcode:
                  0
              stderr:
              stdout:
----------
          ID: strelka_coordinator
    Function: docker_container.running
        Name: so-strelka-coordinator
      Result: True
     Comment: Created container 'so-strelka-coordinator'
     Started: 06:36:07.390791
    Duration: 2478.206 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      b1aed5fc5e1703cce6f93ec1356c9803465fdb2aa9d6ecb3bb81a1ca01261785
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-strelka-coordinator_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 06:36:09.869471
    Duration: 15.193 ms
     Changes:
----------
          ID: strelka_gatekeeper
    Function: docker_container.running
        Name: so-strelka-gatekeeper
      Result: True
     Comment: Created container 'so-strelka-gatekeeper'
     Started: 06:36:09.884925
    Duration: 2114.996 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      0ddf29755aa5be6b76335e00ae5cba0312667289b37331b3108b3ff0624a96d0
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-strelka-gatekeeper_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 06:36:12.000270
    Duration: 3.552 ms
     Changes:
----------
          ID: strelka_frontend
    Function: docker_container.running
        Name: so-strelka-frontend
      Result: True
     Comment: Created container 'so-strelka-frontend'
     Started: 06:36:12.004062
    Duration: 1796.396 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      4d7e054adef9a9e459f292a6e57c39d32b71a27da631df5b507cc7696bb97e08
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-strelka-frontend_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 06:36:13.800800
    Duration: 3.356 ms
     Changes:
----------
          ID: strelka_backend
    Function: docker_container.running
        Name: so-strelka-backend
      Result: True
     Comment: Created container 'so-strelka-backend'
     Started: 06:36:13.804356
    Duration: 1736.759 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      2e22e9eddb1fb6c7aebf043e68ed643847590ccd0be58da3128266af56b1fb58
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-strelka-backend_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 06:36:15.541501
    Duration: 3.13 ms
     Changes:
----------
          ID: strelka_manager
    Function: docker_container.running
        Name: so-strelka-manager
      Result: True
     Comment: Created container 'so-strelka-manager'
     Started: 06:36:15.544847
    Duration: 2255.541 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      b6132364d01b196ee992e7d7b5f1785b387eca1e51b643417929ac0ddd723223
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-strelka-manager_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 06:36:17.802652
    Duration: 13.814 ms
     Changes:
----------
          ID: strelka_filestream
    Function: docker_container.running
        Name: so-strelka-filestream
      Result: True
     Comment: Created container 'so-strelka-filestream'
     Started: 06:36:17.816685
    Duration: 1668.1 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      3170fe5bbb909e3256880709149bb87fcda76bb581a081ae745eaee865e51890
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-strelka-filestream_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 06:36:19.485124
    Duration: 3.314 ms
     Changes:
----------
          ID: strelka_zeek_extracted_sync_old
    Function: cron.absent
        Name: [ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/ > /dev/null 2>&1
      Result: True
     Comment: Cron [ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/ > /dev/null 2>&1 already absent
     Started: 06:36:19.491498
    Duration: 7.402 ms
     Changes:
----------
          ID: strelka_zeek_extracted_sync
    Function: cron.present
        Name: [ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/unprocessed/ > /dev/null 2>&1
      Result: True
     Comment: Cron [ -d /nsm/zeek/extracted/complete/ ] && mv /nsm/zeek/extracted/complete/* /nsm/strelka/unprocessed/ > /dev/null 2>&1 already present
     Started: 06:36:19.499285
    Duration: 8.976 ms
     Changes:

Summary for local
-------------
Succeeded: 25 (changed=7)
Failed:     0
-------------
Total states run:     25
Total run time:   18.410 s


$ sudo so-status

Checking Docker status

    Docker -------------------------------------------------------------------- [ OK ]    
Checking container statuses

    so-cortex ----------------------------------------------------------------- [ OK ]        so-curator ---------------------------------------------------------------- [ OK ]        so-dockerregistry --------------------------------------------------------- [ OK ]        so-elastalert ------------------------------------------------------------- [ OK ]        so-elasticsearch ---------------------------------------------------------- [ OK ]        so-filebeat --------------------------------------------------------------- [ OK ]        so-fleet ------------------------------------------------------------------ [ OK ]        so-grafana ---------------------------------------------------------------- [ OK ]        so-idstools --------------------------------------------------------------- [ OK ]        so-influxdb --------------------------------------------------------------- [ OK ]        so-kibana ----------------------------------------------------------------- [ OK ]        so-kratos ----------------------------------------------------------------- [ OK ]        so-logstash --------------------------------------------------------------- [ OK ]        so-mysql ------------------------------------------------------------------ [ OK ]        so-nginx ------------------------------------------------------------------ [ OK ]        so-playbook --------------------------------------------------------------- [ OK ]        so-redis ------------------------------------------------------------------ [ OK ]        so-sensoroni -------------------------------------------------------------- [ OK ]        so-soc -------------------------------------------------------------------- [ OK ]        so-soctopus --------------------------------------------------------------- [ OK ]        so-steno ------------------------------------------------------------------ [ OK ]        so-strelka-backend ----------------------------------------------------- [ ERROR ]        so-strelka-coordinator ------------------------------------------------- [ ERROR ]        so-strelka-filestream -------------------------------------------------- [ ERROR ]        so-strelka-frontend ---------------------------------------------------- [ ERROR ]        so-strelka-gatekeeper -------------------------------------------------- [ ERROR ]        so-strelka-manager ----------------------------------------------------- [ ERROR ]        so-suricata --------------------------------------------------------------- [ OK ]        so-telegraf --------------------------------------------------------------- [ OK ]        so-thehive ---------------------------------------------------------------- [ OK ]        so-thehive-es ------------------------------------------------------------- [ OK ]        so-wazuh ------------------------------------------------------------------ [ OK ]        so-zeek ------------------------------------------------------------------- [ OK ]

**Comment by licnoya** [comment]
I have this same problem, did you find any solution on how to fix it. Mine occurred after the upgrade to 2.3.50.

I tried restarting services, rebooting the sensor node, but no help.
